package.json

```diff
@@ -96,25 +96,26 @@
     "phantomjs-prebuilt": "^2.1.16",
     "primeicons": "^1.0.0-beta.10",
     "primeng": "^6.0.2",
-    "rxjs": "^6.3.0",
+    "rxjs": "^6.5.2",
     "sinon": "^2.1.0",
     "sinon-chai": "^2.8.0",
     "standard-version": "^4.4.0",
     "ts-node": "^3.0.2",
     "tslint": "^5.11.0",
     "tslint-consistent-codestyle": "^1.14.1",
     "tslint-loader": "^3.5.4",
-    "typescript": "~3.1.1",
+    "typescript": "~3.4.5",
     "validate-commit-msg": "^2.8.0",
-    "webpack": "^2.4.1",
-    "webpack-angular-externals": "^1.0.0",
-    "webpack-dev-server": "^2.2.0",
-    "webpack-rxjs-externals": "^1.0.0",
-    "zone.js": "^0.8.26"
+    "webpack": "^4.33.0",
+    "webpack-angular-externals": "^1.0.2",
+    "webpack-cli": "^3.3.2",
+    "webpack-dev-server": "^3.6.0",
+    "webpack-rxjs-externals": "^2.0.0",
+    "zone.js": "^0.9.1"
```

Comment:
Maybe better to keep these to the old version?

Reply:
I have updated this packages because after I run ng update, the npm start and npm test failed. After update webpack and added webpack-cli, both scripts are successful. When you can run the scripts with the old packages, then it's not necessary.

Reply:
Thank you for this PR.

I'll do some double checks if I can still run the tests and generate the package locally.

BTW: do you think I should just make version `0.8.2` or `0.9.0` or add the Angular version like `8.0.0` ???

Reply:
I think, its not necessary to make a version like angular because your package isn't dependent from exact angular version. I think `0.8.2` or `0.9.0` or `1.0.0` are better.

Reply:
I have tested to revert the 4 packages you are marked, its works when the previous version of `awesome-typescript-loader` (^3.5.0) is installed.